test/spec/controllers/courtSearchInfo.ctrl.spec.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+describe('CourtSearchInfoCtrl', function() {
+  var CourtSearchInfoCtrl;
+  var sample_court;
+
+  sample_court = {
+    id: 9,
+    latitutde: 38.7947506,
+    longitude: -90.26261696,
+    municipality: "Black Jack",
+    address: "12500 Old Jamestown Road",
+    city: "St. Louis",
+    state: "MO",
+    zip_code: 63033
+  };
+
+  beforeEach(function() {
+    module('yourStlCourts');
+
+    inject(function($controller,$state, $window, $q, $httpBackend){
+
+      CourtSearchInfoCtrl = $controller('CourtSearchInfoCtrl',{
+        $state: $state,
+        $window: $window,
+        court:sample_court
+      });
+
+      $httpBackend.whenGET(/courts/).respond(200, '');
+    });
+  });
+
+  xit('goes back to home page if court is null',inject(function($state){
```

Comment:
To test this, you'll need to re-invoke the $controller function with the court as null.
Do this after your ```spyOn($state, 'go')``` and remove the ```sample_court = null``` line:
```CourtSearchInfoCtrl = $controller('CourtSearchInfoCtrl',{
    $state: $state,
    $window: $window,
    court: null
});```

This is because this code gets invoked on controller initialization. You need to essentially create the controller a 2nd time with a different pre-condition than in your beforeEach